vm/src/stdlib/os.rs

```diff
@@ -221,6 +221,37 @@ fn convert_nix_errno(vm: &VirtualMachine, errno: Errno) -> PyClassRef {
     }
 }
 
+// Flags for os_access
+const F_OK: u8 = 0;
+const R_OK: u8 = 4;
+const W_OK: u8 = 2;
+const X_OK: u8 = 1;
```

Comment:
Actually, sorry; this should be a bitflags struct. You can look at `FileCreationFlags` for an example of this, it would probably look something like this:

```rust
bitflags! {
     pub struct AccessFlags: u8 {
        const F_OK = 0;
        const R_OK = 4;
        const W_OK = 2;
        const X_OK = 1;
    }
}
```

Then in `os_access` you can check to see if the user's checking for a certain flag like this:
```rust
let flags = AccessFlags::from_bits(mode)
    .ok_or_else(|| vm.new_value_error("Unsupported flag".to_string()))?;
if flags.contains(AccessFlags::F_OK) {
    ...
}
```
This is all because you can use `access` like this:
```python
os.access(path, R_OK & W_OK)
```

in order to check multiple permissions on the file.